packages/router/src/grpc/view-protocol-server/balances.ts

```diff
@@ -26,8 +26,6 @@ export const balances: Impl['balances'] = async function* (req, ctx) {
       accounts[accountNumber] = {};
     }
 
-    // not throwing ConnectError here, just a normal Error, because these
-    // failures are truly internal. they did not arise from request input.
```

Comment:
this isn't relevant to PR but this comment is incorrect by now and that bothered me